backend/src/server.ts

```diff
@@ -1,37 +1,20 @@
 // Imports secrets from .env file and validates them
-require('./common/utils/env')
+import './common/utils/env'
```

Comment:
To sanity check but I'm not sure of .env validation occurs with import

Reply:
I think since the validation logic is included in the `env.ts`  code and is executed upon import it would.

Reply:
Yup just confirmed, works fine. Had a bug regarding this config in an old project, but I think it was a misconfiguration with the typescript compiling tooling I had setup. Great refactor